docs/src/main/asciidoc/ghpages.sh

```diff
@@ -145,6 +154,17 @@ function copy_docs_for_current_version() {
             fi
         done
         COMMIT_CHANGES="yes"
+    elif [[ ",${BETA_RELEASE_BRANCHES_VALUE}," = *",${CURRENT_BRANCH},"* ]] ; then
+        echo -e "Branch [${CURRENT_BRANCH}] is a whitelisted beta release branch! Will copy the current docs to the root folder"
+        for f in docs/target/generated-docs/*; do
+            file=${f#docs/target/generated-docs/*}
+            if ! git ls-files -i -o --exclude-standard --directory | grep -q ^$file$; then
+                # Not ignored...
+                cp -rf $f ${ROOT_FOLDER}/
+                git add -A ${ROOT_FOLDER}/$file
+            fi
+        done
+        COMMIT_CHANGES="yes"
     else
         echo -e "Current branch is [${CURRENT_BRANCH}]"
```

Comment:
If we always use `${version}` instead of `${branch}/${version}`, we can delete all logic about `${branch}/${version}.`